wrapper/src/main/java/software/amazon/jdbc/plugin/federatedauth/NonValidatingFactory.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.plugin.federatedauth;
+
+import java.security.GeneralSecurityException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * Provide a SSLSocketFactory that allows SSL connections to be made without validating the server's
+ * certificate. This is more convenient for some applications, but is less secure as it allows "man
+ * in the middle" attacks.
+ */
+public class NonValidatingFactory extends WrappedFactory {
```

Comment:
Too vague of a name... what about NonValidatingSSLSocketFactory

Reply:
Going with NonValidatingSSLSocketFactory as suggested